### DISCUSSION TITLE
Vite or CRA?

### DISCUSSION THREAD

**Discussion by MuhammadRabi** [chingu-voyages/v43-tier2-team-14#15] [Polls]
Hi guys! do you prefer Vite or Create React App for our project?

https://vitejs.dev/

https://create-react-app.dev/

**Comment by MuhammadRabi** [comment]
for me Vite is way faster and lighter than CRA

**Reply by ahmed-lotfy-dev** [reply]
Better, lighter, and faster eventually we'll see what the poll result
